react/src/hooks/projects/useProjects.ts

```diff
@@ -75,3 +75,17 @@ export function useProjectsWithDesignSafeInformation(): UseQueryResult<
     error: dsProjectQuery.error || projectQuery.error,
   } as UseQueryResult<Project[]>;
 }
+
+export const useDeleteProject = (projectId: number | undefined) => {
```

Comment:
```suggestion
export const useDeleteProject = (projectId: number) => {
```

this could be changed if we fix the issue with optional members in https://github.com/TACC-Cloud/hazmapper/pull/273#discussion_r1807034709 (and then some other changes in its usage).